packages/babel-config/src/plugins/babel-plugin-redwood-routes-auto-loader.ts

```diff
@@ -149,6 +148,7 @@ export default function (
           for (const { importName, relativeImport } of pages) {
             const importArgument = t.stringLiteral(relativeImport)
 
+            // TODO(jgmw): Determine if we still need these `webpackChunkName` comments or not
```

Comment:
We don't need this anymore. 

The comments would cause webpack to generate chunks with the comment's filename so that we can insert the correct chunk during prerender.

e.g. /about -> would insert /about.xxx.js